main.py

```diff
@@ -1,12 +1,197 @@
 import requests
-
-URL = "http://127.0.0.1:5000"
-BACKUP_URL = "https://retro-video-store-api.herokuapp.com"
+URL = "https://retro-video-store-api.herokuapp.com"
 
 def main():
-    print("WELCOME TO RETRO VIDEO STORE")
     pass
 
-
 if __name__ == "__main__":
-    main()
\ No newline at end of file
+    main()
+
+from requests.models import to_native_string
+from retro_video import RetroVideo
+
+def print_retro_video():
+    print("\n⭐⭐⭐ Retro 📼 Video ⭐⭐⭐\n")    
+
+def print_stars():
+    print("\n⭐⭐⭐⭐⭐⭐⭐⭐⭐⭐⭐⭐⭐⭐⭐⭐⭐⭐\n")
+
+def list_options():
+    options = {
+        "1": "add a video",
+        "2": "edit a video",
+        "3": "delete a video",
+        "4": "get information about all videos",
+        "5": "get information about one video",
+        ######################################
+        "6": "add a customer",
+        "7": "edit a customer",
+        "8": "delete a customer",
+        "9": "get information about one customer",
+        "10": "get information about all customers",
+        ######################################
+        "11": "check out a video to a customer",
+        "12": "check in a video from a customer",
+        "13": "Quit"
+    }
+
+    print_retro_video()
+    print("Welcome to Retro Video CLI")
+    print("These are the actions you can perform")
+    print_stars()
+
+    for choice_num in options:
+        print(f"Option {choice_num}. {options[choice_num]}")
+    print_stars()
+
+    return options
+
+def make_choice(options, retro_video):
+    valid_choices = options.keys()
+    choice = None
+    while choice not in valid_choices:
+        print("What would you like to do?")
+        choice = input("Make your selection using the option number: ")
+    return choice 
+
+def run_cli(play=True):
+    #initialize retro_video
+    retro_video = RetroVideo(url="https://retro-video-store-api.herokuapp.com/")
+
+    options = list_options()
+
+    while play==True:
+        choice = make_choice(options, retro_video)
+
+                #######################################
+                ########### VIDEO OPTIONS #############
+                #######################################
+### CHOICE 1 ###
+        if choice=="1": # add a video
+            print("Great! Let's add a new video.")
+            title=input("What is the title of the video? ")
+            release_date=input("What is the release date of the video? ")
+            total_inventory=input("What is the total inventory of the video?")
+            response = retro_video.create_video(title=title, release_date=release_date, total_inventory=total_inventory)
+            print_stars()
+            print(f"The New Video \'{title}\' with {response} was successfully added.")
+### CHOICE 2 ###
+        elif choice=="2": # edit a video
+            retro_video.select_video()
```

Comment:
If the video selection failed (ie, if the user attempted to get an ID that doesn't exist in the database) and the selected_video is None, the following code will result in an exception.